powerbi-docs/developer/visuals/create-bar-chart.md

```diff
@@ -200,13 +197,13 @@ Replace the content of the "objects" section with the following:
 
 Save the *capabilities.json* file.
 
-Your final *capabilities* file should look like [the one in this example](https://github.com/blackleaden/PowerBI-visuals-sampleBarChart/blob/barChartTutorial/capabilities.json).
```

Comment:
where did that come from ??!?!?!? (backleaden)

Reply:
Seems to be a worker in microsoft-akvelon